amethyst_renderer/Cargo.toml

```diff
@@ -36,7 +36,7 @@ gfx_core = { version = "0.8", features = ["serialize"] }
 gfx_macros = "0.2"
 imagefmt = "4.0"
 log = "0.4"
-rayon = "0.8"
+rayon = "1.0"
```

Comment:
Need to bump version of `hetseq` to `0.2` here so it also depend on `rayon` `1.0`.

Reply:
I can, but note the `hetseq/rayon` feature is not enabled anyway.

Reply:
Indeed 😄 Then you may skip this request